.github/workflows/backlogger.yaml

```diff
@@ -3,7 +3,6 @@ name: Backlog Limits Checker Preview
 # yamllint disable-line rule:truthy
 on:
   push:
-    branches: ['main']
```

Comment:
This needs to stay there. We only want to deploy for commits on the main branch.